content/docs/api-reference/app-registrations-api/index.md

```diff
@@ -74,24 +69,40 @@ The client secret ***must*** be taken note of at this point, as it will be redac
 }
 ```
 
+{{< caution >}}
+**Caution:** The expiry date refers to the secret only, any tokens generated with this secret will not automatically become invalid when the secret expires or is rotated. Each token has a TTL of 1 hour by default. 
+{{< /caution >}}
+
 #### Authenticating with your Application
 
-Now that you've created an application, the `client_id` and `secret` can be used to obtain a bearer token:
+Now that you've created an application, you get a token.
+
+Replace `${CLIENT_ID}` with the application id, and `${SECRET}` with your secret from the application registration.
 
 ```bash
-export CLIENT_ID=d1fb6c87-faa9-4d56-b2fd-a5b70a9af065
-export SECRET=a0c09972b6ac912a4d67815fef88093c81a99b49977d35ecf6d162631aa29173
-
-TOKEN=$(curl -X POST -H "Content-Type: application/x-www-form-urlencoded" \
-     --url ${URL}/archivist/iam/v1/appidp/token \
-     --data-urlencode "grant_type=client_credentials" \
-     --data-urlencode "client_id=${CLIENT_ID}" \
-     --data-urlencode "client_secret=${SECRET}") && echo "Authorization: Bearer ${TOKEN}" > app-id.jwt
+curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
+    --data-urlencode "grant_type=client_credentials" \
+    --data-urlencode "client_id=${CLIENT_ID}" \
+    --data-urlencode "client_secret=${SECRET}"
 ```
 
-You can then use this bearer token to interact with other RKVST services.
```

Comment:
Ah good to have the link to the standard. I would appreciate that as a developer